.semaphore/semaphore.yml.d/blocks/20-typha.yml

```diff
@@ -11,6 +11,12 @@
       - name: "Typha: UT and FV tests"
         commands:
           - ../.semaphore/run-and-monitor make-ci.log make ci EXCEPT=k8sfv-test
+      - name: "Typha: UT and FV tests on UBI-minimal"
+        commands:
+          - ../.semaphore/run-and-monitor make-fv-ubi.log make clean image fv
+        env_vars:
+          - name: CALICO_BASE
+            value: registry.access.redhat.com/ubi8/ubi-minimal
```

Comment:
Could you make this is a special "use ubi" value and do the switch to the exact image in the makefile?  Just thinking that we already have `registry.access.redhat.com/ubi8/ubi-minimal` in metadata.mk

Reply:
Done